articles/tables.md

```diff
@@ -164,8 +162,8 @@ If you upload a file, the Web interface will automatically detect the table sche
 The Table() function takes two arguments, a schema object and data in some form, which can be:
 
 - a path to a CSV file
-- a [Pandas](http://pandas.pydata.org/){:target="_blank"} [DataFrame](http://pandas.pydata.org/pandas-docs/stable/api.html#dataframe){:target="_blank"} (This is a Python-only feature)
-- a `RowSet` object
+- a dataframe
+- a [`RowSet` object](http://docs.synapse.org/rest/org/sagebionetworks/repo/model/table/RowSet.html){:target="_blank"}
 - a list of lists where each of the inner lists is a row
```

Comment:
not sure this is accurate (though I see you got it from the Synapse Python docstring)